src/repositories/commits/lib/fillAuthorModel.ts

```diff
@@ -0,0 +1,16 @@
+import { Model } from 'mongoose';
+import { UserDocument } from '../../../model/schemas';
+import { Commit } from '../model';
+
+export async function fillAuthorModel(
+  commits: Commit[],
+  userModel: Model<UserDocument>,
+) {
+  const filledCommits: Commit[] = [];
+  for (const commit of commits) {
+    const author = await userModel.create(commit.author);
```

Comment:
move the existing check to this method and keep using it? else remove it completly